static-assets/components/cstudio-forms/controls/rte.js

```diff
@@ -354,20 +314,13 @@ CStudioAuthoring.Module.requireModule(
               imageDatasources.includes(datasource.id)
           );
 
-          const codeEditorWrap = rteConfig.codeEditorWrap ? rteConfig.codeEditorWrap === 'true' : false;
-
           const external = {
+            ...rteConfig?.tinymceOptions?.external_plugins,
```

Comment:
Why would rteConfig ever be nullish? This shouldn't run prior to the config initialization.

Reply:
It's never going to be null, removing the optional chaining. This function is called after initialization so we're good with that.